src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
Hm. My initial reaction was, "I have no guarantee that `spawn` isn't going to be used in other cross-cutting ways by future `distutils` code in later Py3 revisions, while I have a pretty strong semver guarantee that I'm safe until Py4 since I'm maintaining the semantics that're now present in Py3 for this function."

Maybe code simplicity and maintenance comfort is preferable, though. While I'm not quite yet on the fence about it, I'm leaning rather heavily on a weak board and I can hear some rusted nails popping.